package.json

```diff
@@ -56,50 +56,52 @@
   },
```

Comment:
I don't know much about this, but when WebOps builds for production, doesn't it already [skip](https://classic.yarnpkg.com/en/docs/cli/install/#toc-yarn-install-production-true-false) all the `devDependencies` because `NODE_ENV` is set to `production` (I'd hope)?

Reply:
@filipewl It does not because it has to run the tests and the lighthouse checks.

Reply:
But for building the static pages, I don't think they need to install the dev dependencies...

The tests and LH checks are run on their own separate pipeline run and in these cases they do install the dev dependencies to run the proper commands.

Reply:
@filipewl, indeed the devDependencies are not needed for building the static pages, as well as the tests and LH checks have separate pipelines.

However, if you look at our tekton dashboard and browse the pipelines' code, you'll notice that they share the same common task, called `clone`. This `clone` task is responsible for, among other things, running `yarn install` for the cloned repo in the CI environment. Since this is shared between pipelines, we need to fill the `optionalDependencies` with the packages used in all pipelines, letting out only things we use locally, such as the `storybook` and `eslint`.

I agree that different tasks like `clone-for-test` and `clone-for-build` would be better, but:
- `yarn` does not support other sections besides `optionalDependencies`. If they had support for `testDependencies` and `buildDependencies` that would be great.
- The impact on the time is minimal as per the benchmarks I've run (adding cypress and lighthouse dependencies adds ~4s comparing to when only using the build dependencies)

We could investigate alternatives about to reduce the install time even further, maybe at webops level we could run different `yarn install` commands depending on the pipeline. But that is a story for another PR :)